File: mini-commerce/src/main/java/me/hero/minicommerce/item/adapter/in/web/dto/FindItemResponse.java
package me.hero.minicommerce.item.adapter.in.web.dto;

import lombok.Data;
import me.hero.minicommerce.item.application.port.in.dto.FindItemDto;

@Data
public class FindItemResponse {
  private final String name;
  private final Long price;

  public FindItemResponse(FindItemDto itemDto) {
    this.name = itemDto.getName();
    this.price = itemDto.getPrice();
  }
}

File: mini-commerce/src/main/kotlin/me/hero/minicommerce/item/adapter/in/web/dto/FindItemResponse.kt
package me.hero.minicommerce.item.adapter.`in`.web.dto

import me.hero.minicommerce.item.application.port.`in`.dto.FindItemDto

data class FindItemResponse(
    val name: String,
    val price: Long,
) {
    constructor(itemDto: FindItemDto) : this(
        name = itemDto.name,
        price = itemDto.price,
    )
}